catalog/lib/theme/app_color_scheme.dart

```diff
@@ -88,8 +86,8 @@ class AppColorScheme extends ColorScheme {
             },
           ),
           onError: Colors.black,
-          onBackground: const Color(0xffadadad),
-          background: const Color(0xfff7fafd),
+          onSurface: const Color(0xffadadad),
+          surface: const Color(0xfff7fafd),
```

Comment:
do we have to delete background and onBackground?

Reply:
`onBackground` and `background` is deprecated in the latest version
https://api.flutter.dev/flutter/material/ColorScheme/onBackground.html